pom.xml

```diff
@@ -78,6 +78,11 @@
       <artifactId>dnsjava</artifactId>
       <version>2.1.8</version>
     </dependency>
+    <dependency>
+	  <groupId>commons-logging</groupId>
```

Comment:
I don't see this dependency used in the code - do we need it?

Reply:
HttpGet was using this internally and build was failing without this dependency. Hence i added it